tests/utils/series-utils-tests.js

```diff
@@ -100,7 +100,7 @@ test('series-utils #seriesClusterProps', t => {
   t.end();
 });
 
-test('series-utils #getStackedData', t => {
+test('series-utils #getStackedData', t => { // eslint-disable-line max-statements
```

Comment:
Wrong commit message here, sorry about that. 
For the eslint comment, I hope it's ok to disable only that rule here...

Reply:
Very reasonable, probably would also be reasonable to set that for all of the test suite, but that doesn't need to happen here